.gitlab-ci.yml

```diff
@@ -92,13 +92,14 @@ analysis:licenses:
   script:
     - GRADLE_OPTS="-Xmx2560m" ./gradlew checkThirdPartyLicensesAll --stacktrace --no-daemon
 
-analysis:api-surface:
-  tags: [ "runner:main" ]
-  image: $CI_IMAGE_DOCKER
+static-analysis:
   stage: analysis
   timeout: 30m
-  script:
-    - GRADLE_OPTS="-Xmx2560m" ./gradlew checkApiSurfaceChangesAll --stacktrace --no-daemon
+  variables: 
+    GRADLE_OPTS: "-Xmx2560m -Dorg.gradle.daemon=false -Porg.gradle.logging.stacktrace=full"
```

Comment:
🤔 Should we let the template consumer setup the gradle configuration or should it be defined at the shared gitlab template level.
One issue we have here is we define the env variables for all jobs defined and run from the gitlab templates
(same thing regarding the `timeout` gitlab param)